src/main/java/de/theit/jenkins/crowd/CrowdConfigurationService.java

```diff
@@ -420,9 +446,12 @@ public User getUser(String username) throws UserNotFoundException, OperationFail
         // Load the entry from cache if it's valid return it
         User retval = getValidValueFromCache(username, userCache);
         if (retval != null) {
+            LOG.log(Level.FINEST, "getUser() cache hit: {0}", username);
             return retval;
         }
 
+        LOG.log(Level.FINEST, "getUser() cache hit MISS: {0}", username);
+
```

Comment:
Please change this log to debug. Mostly because it could be pretty extensive in bigger organization.
Also there is another log in L455, it's better to concatenate both logs